bitnami/harbor/templates/jobservice/jobservice-dpl.yaml

```diff
@@ -0,0 +1,114 @@
+      - name: jobservice
+        image: "{{ template "harbor.jobserviceImage" . }}"
+        imagePullPolicy: {{ .Values.jobserviceImage.pullPolicy | quote }}
+        {{- if .Values.jobservice.livenessProbe.enabled }}
+        livenessProbe:
+          httpGet:
+            path: /api/v1/stats
+            port: 8080
+          initialDelaySeconds: {{ .Values.jobservice.livenessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.jobservice.livenessProbe.periodSeconds }}
+          timeoutSeconds: {{ .Values.jobservice.livenessProbe.timeoutSeconds }}
+          successThreshold: {{ .Values.jobservice.livenessProbe.successThreshold }}
+          failureThreshold: {{ .Values.jobservice.livenessProbe.failureThreshold }}
+        {{- end }}
+        {{- if .Values.jobservice.readinessProbe.enabled }}
+        readinessProbe:
+          httpGet:
+            path: /api/v1/stats
+            port: 8080
+          initialDelaySeconds: {{ .Values.jobservice.readinessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.jobservice.readinessProbe.periodSeconds }}
+          timeoutSeconds: {{ .Values.jobservice.readinessProbe.timeoutSeconds }}
+          successThreshold: {{ .Values.jobservice.readinessProbe.successThreshold }}
+          failureThreshold: {{ .Values.jobservice.readinessProbe.failureThreshold }}
+        {{- end }}
+{{- if .Values.jobservice.resources }}
+        resources:
+{{ toYaml .Values.jobservice.resources | indent 10 }}
+{{- end }}
+        env:
+          {{- if .Values.jobserviceImage.debug }}
+          - name: BITNAMI_DEBUG
+            value: "true"
+          {{- end }}
+          - name: CORE_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: {{ template "harbor.core" . }}
+                key: secret
+          - name: JOBSERVICE_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: "{{ template "harbor.jobservice" . }}"
+                key: secret
+          - name: CORE_URL
+            value: "http://{{ template "harbor.core" . }}"
+          - name: REGISTRY_CONTROLLER_URL
+            value: "http://{{ template "harbor.registry" . }}:8080"
+          - name: LOG_LEVEL
+            value: debug
+        ports:
+        - containerPort: 8080
+        volumeMounts:
+        - name: jobservice-config
+          mountPath: /etc/jobservice/config.yml
+          subPath: config.yml
+        - name: job-logs
+          mountPath: /var/log/jobs
+          subPath: {{ .Values.persistence.persistentVolumeClaim.jobservice.subPath }}
+      volumes:
+      - name: jobservice-config
+        configMap:
+          name: "{{ template "harbor.jobservice" . }}"
+      - name: job-logs
+        {{- if and .Values.persistence.enabled (eq .Values.jobservice.jobLogger "file") }}
+        persistentVolumeClaim:
+          claimName: {{ .Values.persistence.persistentVolumeClaim.jobservice.existingClaim | default (include "harbor.jobservice" .) }}
```

Comment:
Add a note that the volume must be ReadWriteMany or it won't scale. I see that you allow more replicas

Reply:
Done